projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -938,4 +946,44 @@ export class IgxOverlayService implements OnDestroy {
             info.closeAnimationPlayer.finish();
         }
     }
+
+    /**
+     * @hidden
+     */
+    public proxy = new Proxy(
```

Comment:
the proxy should not be exposed to the outside world. Only the overlay should know of the existence of the Proxy.